### ISSUE TITLE
luarocks packaging doesn't include fennelview

### ISSUE THREAD

**Issue by technomancy** [bakpakin/Fennel#120]
Would be nice to cut a 0.1.1 bugfix release with the most recent fixes where luarocks users could have fennelview in their repl by default.

@bakpakin what kind of problems were you running into when you tried this previously?

**Comment by the-real-molloy** [comment]
May have found the cause— [the Luarocks docs](https://github.com/luarocks/luarocks/wiki/Rockspec-format#builtin) state that string values in the `modules` array should be pathnames of Lua files or C sources, so compiling `fennelview.fnl` and including `fennelview.lua` as a module should do the trick. [Previously](https://github.com/bakpakin/Fennel/blob/e54a85b3525a44ac16d6a4e35d19a1d5d6948ce2/rockspecs/fennel-0.1.0-1.rockspec#L22) it was being included as `fennelview.fnl`.

**Comment by bakpakin** [comment]
Yeah, that was the problem. What confused me was that it was there in the first place, so it may have worked in the past.

I know that the moonscript compiler is written in moonscript, but the rock contains Lua versions of all of the files.

**Comment by bakpakin** [comment]
Alright, I have released a new version 0.1.1 that should have fennelview included.
Following moonscript, I have included the compiled output in the repository as it makes releasing easier.

Please try installing fennel now and ensure you get the expected output.
